src/winetricks

```diff
@@ -6912,14 +6912,14 @@ helper_dxvk()
         w_try_unzip "${W_TMP}" "${W_CACHE}/${W_PACKAGE}/${_W_package_archive}"
     else
         w_try_cd "${W_TMP}"
-        w_try tar -zxf "${W_CACHE}/${W_PACKAGE}/${_W_package_archive}"
+        w_try tar -zxf "${W_CACHE}/${W_PACKAGE}/${_W_package_archive}" --one-top-level
```

Comment:
`--one-top-level` is a GNU extension; this will break on BSD tar.

Reply:
Thanks! After some light research it looks like `-C` might work for BSD tar but I haven't tested just yet.

Reply:
Updated to use `-C "${_W_package_dir}"`, Winetricks still fetches and installs d8vk correctly (tested with no cached archive and with a cached archive), only tested Touhou 6 at the moment and it still works too.